install_chromedriver.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+CHROME_VERSION="$(dpkg-query -W -f='${source:Upstream-Version}' google-chrome-beta)"
+
+echo "Installing ChromeDriver $CHROME_VERSION"
+
+wget -q "https://chromedriver.storage.googleapis.com/$CHROME_VERSION/chromedriver_linux64.zip"
```

Comment:
Is this URL guaranteed to exist?

Reply:
:shrug: no idea, the pattern seems to be that builds for beta versions are uploaded, but I don't know if we can depend on that.

Reply:
Ok let’s see how it goes.